apps/mobile/app/screens/verify-email.tsx

```diff
@@ -0,0 +1,650 @@
+import React, { useState, useEffect, useRef } from 'react';
+import {
+  View,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  KeyboardAvoidingView,
+  Platform,
+  StyleSheet,
+  useWindowDimensions,
+  ScrollView,
+  Animated,
+} from 'react-native';
+import { useRouter, useLocalSearchParams } from 'expo-router';
+import Svg, { Path } from 'react-native-svg';
+
+export default function VerifyEmailScreen() {
+  const router = useRouter();
+  const { email = "user@example.com" } = useLocalSearchParams<{ email: string }>();
+  const { width, height } = useWindowDimensions();
+  
+  const isSmallDevice = width < 375;
+  const isLargeDevice = width > 414;
+  const isLandscape = width > height;
+  
+  const [code, setCode] = useState<string[]>(['', '', '', '', '', '']);
```

Comment:
Cuando definís el default value de un state, TypeScript ya infiere de una el tipo de este state, no hace falta definirlo explícitamente
```suggestion
  const [code, setCode] = useState(['', '', '', '', '', '']);
```